racun-service/src/main/java/rs/edu/raf/banka/racun/service/impl/RacunService.java

```diff
@@ -13,12 +15,14 @@
 public class RacunService {
 
     private RacunRepository racunRepository;
+    private ValutaRepository valutaRepository;
     private final SredstvaKapitalService sredstvaKapitalService;
 
     @Autowired
-    public RacunService(RacunRepository racunRepository, SredstvaKapitalService sredstvaKapitalService){
+    public RacunService(RacunRepository racunRepository, SredstvaKapitalService sredstvaKapitalService, ValutaRepository valutaRepository){
         this.racunRepository = racunRepository;
         this.sredstvaKapitalService = sredstvaKapitalService;
+        this.valutaRepository = valutaRepository;
     }
 
     public Racun createRacun(){
```

Comment:
Vrati ovu metodu kako je bilo i nazovi je createKesRacun
Ovo neka bude u metodi:
        Racun racun = new Racun();
        racun.setBrojRacuna(UUID.randomUUID());
        racun.setTipRacuna(RacunType.KES);
        racunRepository.save(racun);
        sredstvaKapitalService.pocetnoStanje(racun.getBrojRacuna(), "RSD", 100000);
        sredstvaKapitalService.pocetnoStanje(racun.getBrojRacuna(), "USD", 100000);
        sredstvaKapitalService.pocetnoStanje(racun.getBrojRacuna(), "EUR", 100000);

        return racun;

Napravi jos jednu metodu createMarginRacun u kojoj cemo da inicijalizujemo marginRacun pozivajuci metodu iz SredstvaKapitalService, slicno kao createKesRacun

Reply:
Treba nam takva metoda i za maržni račun, na primer `createMarginsAccount`. Samo sto tu pozivaš onu novu metodu što si dodao.